API_compatible_README.md

```diff
@@ -0,0 +1,24 @@
+# API Compatible
+
+*Instructions are for setting up API for Audacious client on Ubuntu. The steps can be modified for other clients as well.*
+
+**SOURCE_1**: http://redmine.audacious-media-player.org/boards/1/topics/788
+
+1. Install dependencies from **SOURCE_1** then clone the repo and install audacious.
+2. Before installing audacious-plugins, edit the file `audacious-plugins/src/scrobbler2/scrobbler.h` to update the following settings on lines L26-L28.  
+  - `SCROBBLER_API_KEY` to the key given on "listenbrainz.org/user/import" page.
+  - `SCROBBLER_URL` to "http://last.fm/2.0/".  
```

Comment:
One of the reasons that we want to have a compatibility API is so that users can submit listens to ListenBrainz without having to change any code or recompile their software (consider users of windows software who can only download a binary, or even linux users who install from their package manager)
In fact, if they are changing code, then there would be nothing stopping us from changing the scrobbler url to listenbrainz.org, for example.

Why do you require that the API key here is set to the user's import page? From my understanding of lastfm authentication, this key is specific to the application, not to each individual user.